src/ripple/rpc/impl/RPCHelpers.cpp

```diff
@@ -358,13 +358,13 @@ Status
 getLedger(T& ledger, LedgerShortcut shortcut, Context& context)
 {
     if (isValidatedOld(context.ledgerMaster, context.app.config().standalone()))
-        return {rpcNO_NETWORK, "InsufficientNetworkMode"};
+        return {rpcNOT_SYNCED, "InsufficientNetworkMode"};
```

Comment:
Is there a point to returning a text string that's different from the description of `rpcNOT_SYNCED` here?

Reply:
Good catch.  Fixed this in several places in RPCHelpers.cpp